ocfweb/docs/views/servers.py

```diff
@@ -56,96 +57,93 @@ def english_type(self):
     def has_munin(self):
         return self.type in ('hypervisor', 'vm', 'server', 'desktop')
 
+    def __key(self):
+        """Key function used for comparison."""
+        ORDER = {
+            'hypervisor': 1,
+            'server': 2,
+            'desktop': float('inf'),
+        }
+        DEFAULT_ORDER = 3
+        return (ORDER.get(self.type, DEFAULT_ORDER), self.type, self.hostname)
```

Comment:
all upper case is a style for global constants. leave this as `ranking` and `default` or something like that.